### PR TITLE
Update organisations to use STI

### PR DESCRIPTION
**Pull request by cesidio**
This make School and SchoolGroup become subclasses of Organisation model.

### CODE REVIEW

File: lib/tasks/create_organisations.rake [hunk 1]
```diff
@@ -11,5 +11,11 @@ namespace :data do
         Organisation.create(school_group.attributes)
       end
     end
+
+    desc 'Update organisations data'
```
Comment (alexbowen) [inline lib/tasks/create_organisations.rake:15]: indentation looks slightly out here?

**Comment by alexbowen** [comment]
whilst code looks pretty straightfoward change, could you add description as i dont what its supposed to acheive?

File: db/migrate/20200813105100_remove_foreign_keys.rb [hunk 2]
```diff
@@ -0,0 +1,6 @@
+class RemoveForeignKeys < ActiveRecord::Migration[5.2]
+  def change
+    remove_foreign_key :user_preferences, :school_groups
+    remove_foreign_key :vacancies, :school_groups
```
Comment (david-mears-dfe) [inline db/migrate/20200813105100_remove_foreign_keys.rb:4]: What replaces these removed foreign keys?

Comment (cesidio) [reply db/migrate/20200813105100_remove_foreign_keys.rb:4]: Nothing for now

File: lib/tasks/create_organisations.rake (hunk 1 above)
Comment (cesidio) [reply lib/tasks/create_organisations.rake:15]: Indentation is correct here, it has to be aligned to the task

**Pull request merged**